util/src/main/java/io/kubernetes/client/util/PagerParams.java

```diff
@@ -0,0 +1,30 @@
+package io.kubernetes.client.util;
```

Comment:
It's weird that this is in a different package than `Pager`. move to `pager`? or `util.pager`

Reply:
moved to pager package.